RecoParticleFlow/PFClusterProducer/plugins/PFClusterProducer.cc

```diff
@@ -21,6 +21,16 @@ PFClusterProducer::PFClusterProducer(const edm::ParameterSet& conf)
     const std::string& cleanerName = conf.getParameter<std::string>("algoName");
     _cleaners.emplace_back(RecHitTopologicalCleanerFactory::get()->create(cleanerName, conf));
   }
+
+  if (conf.exists("seedCleaners")) {
```

Comment:
This ```conf.exists(...)``` is put here just to avoid crashes in the HLT during the limited period of time in which this updated code is not yet parsed in the menu.
    a) The optimal way of dealing with it would be to add a ```fillDescription()``` method to the PFClusterProducer. This would require major changes in the code, and I think such an update could go well beyond the scope of this PR
    b) In alternative, one could better avoid using ```conf.exists(...)``` as it is now in the code, and rather customize temporarily the HLT menu in ```HLTrigger/Configuration/python/customizeHLTforCMSSW.py```

Since there are other requests to touch this code again, perhaps you can take this into account and get rid of this ```conf.exists(...)``` already in this PR. Alternatively, this could be deferred to a forthcoming PR